### PR TITLE
A couple of assorted tweaks

### PR DESCRIPTION
**Pull request by mrc0mmand**
Prompted by a couple of recent-ish CI fails (still going through the backlog).

### CODE REVIEW

**Review by yuwata — APPROVED** [review]

**Comment by mrc0mmand** [comment]
```
Finishing after writing 35785 entries
TEST-54-CREDS: [    6.569956] H systemd-executor[433]: Failed to deserialize context: Invalid argument
-rw-r----- 1 root root 25165824 Apr  2 15:47 /var/tmp/systemd-test-TEST-54-CREDS/system.journal
qemu test failed
make: *** [Makefile:4: run] Error 1
make: Leaving directory '/build/test/TEST-54-CREDS'
```

Hehe, looks like the check is working.

File: test/units/end.sh [hunk 1]
```diff
@@ -9,5 +9,9 @@ set -o pipefail
 # Here, the redundant '[ ]' in the pattern is required in order not to match the logged command itself.
 (! journalctl -q -o short-monotonic --grep 'Warning: cannot close sd-bus connection[ ].*after fork' >>/failed)
 
+# Check if sd-executor doesn't complain about not being able to (de)serialize stuff
+(! journalctl -q -o short-monotonic --grep "[F]ailed to parse serialized line" >>/failed)
```
Comment (dnicolodi) [inline test/units/end.sh:13]: What is the reason for the `[F]` construct?

Comment (mrc0mmand) [reply test/units/end.sh:13]: The test scripts run with `set -x` and are eventually logged into the journal. The `[F]` stuff is necessary so the grep doesn't match the executed command itself.

File: test/units/testsuite-74.coredump.sh [hunk 2]
```diff
@@ -77,31 +77,35 @@ rm -fv /run/systemd/coredump.conf.d/99-external.conf
...
+    timeout "$TIMEOUT" bash -xec "until systemd-run -M '$CONTAINER' -q --wait --pipe true; do sleep .5; done"
 
     [[ "$(systemd-run -M "$CONTAINER" -q --wait --pipe coredumpctl list -q --no-legend /usr/bin/sleep | wc -l)" -eq 0 ]]
     machinectl copy-to "$CONTAINER" "$MAKE_DUMP_SCRIPT"
     systemd-run -M "$CONTAINER" -q --wait --pipe "$MAKE_DUMP_SCRIPT" "/usr/bin/sleep" "SIGABRT"
     systemd-run -M "$CONTAINER" -q --wait --pipe "$MAKE_DUMP_SCRIPT" "/usr/bin/sleep" "SIGTRAP"
     # Wait a bit for the coredumps to get processed
     timeout 30 bash -c "while [[ \$(systemd-run -M $CONTAINER -q --wait --pipe coredumpctl list -q --no-legend /usr/bin/sleep | wc -l) -lt 2 ]]; do sleep 1; done"
+
+    rm -rf "/var/lib/machnies/$CONTAINER"
```
Comment (bluca) [inline test/units/testsuite-74.coredump.sh]: typo

File: test/units/testsuite-74.coredump.sh [hunk 3]
```diff
@@ -77,31 +77,35 @@ rm -fv /run/systemd/coredump.conf.d/99-external.conf
...
 
-# Make sure we can forward crashes back to containers
-CONTAINER="testsuite-74-container"
-
-mkdir -p "/var/lib/machines/$CONTAINER"
-mkdir -p "/run/systemd/system/systemd-nspawn@$CONTAINER.service.d"
-# Bind-mounting /etc into the container kinda defeats the purpose of --volatile=,
-# but we need the ASan-related overrides scattered across /etc
-cat > "/run/systemd/system/systemd-nspawn@$CONTAINER.service.d/override.conf" << EOF
+if cgroupfs_supports_user_xattrs; then
```
Comment (bluca) [inline test/units/testsuite-74.coredump.sh:80]: This change is not mentioned in the commit message

**Review by bluca — APPROVED** [review]

**Pull request merged**